src/Table.php

```diff
@@ -502,13 +502,20 @@ public function renderRow(): void
      * click outside of the body. Additionally when you move cursor over the
      * rows, pointer will be used and rows will be highlighted as you hover.
      *
-     * @param JsChain|\Closure|JsExpressionable $action Code to execute
+     * @param JsChain|\Closure|JsExpressionable $action          Code to execute
+     * @param array<int, string>                $excludeSelector CSS classes to exclude from row click
      */
     public function onRowClick($action): void
     {
         $this->addClass('selectable');
         $this->js(true)->find('tbody')->css('cursor', 'pointer');
 
+        /* Do not trigger row click event if click stems from row content
+         like checkboxes, or links, marked as atk4-norowclick
+         */
+        $this->js(true )->find('.atk4-norowclick')->on('click',
```

Comment:
well, why do we need a selector then :) (in chat, you have written `:not(...)` class is not working)

Reply:
I do not understand - we need to stopPropagation for Link, or Checkbox, so we flag them with the `atk4-norowclick` class.

Reply:
component level (checkbox, link, ...) is responsible to prevent such events, why `event.stopPropagation` is then setup from a different component?

I see the exclusions are setup `src/Table/Column/*` and this kinda makes sense then...

but this is again a Table/Grid/Crud specific fix and CardCard sail out of sync again :)

let's start with checkbox click test

then I am fine to merge the current fix as for Table components it LGTM